modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -49,6 +49,53 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
         }
       }
     }
+
+    // Text Background Color.
+    if (!empty($this->configuration['bg_text_color'])) {
+      $bg_text_color = $row->getSourceProperty($this->configuration['bg_text_color']);
+      $bg_text_color_mapping = [
+        'bg-transparent' => 'transparent',
+        'bg-trans-white' => 'light',
+        'bg-trans-sky' => 'light',
+        'bg-trans-arizona-blue' => 'dark',
+        'bg-trans-black' => 'dark',
+      ];
+      foreach ($bg_text_color as $bg_text_color_item) {
+        $behavior['bg_color'] = $bg_text_color_mapping[$bg_text_color_item['value']];
+      }
+    }
+
+    // Text Background Color.
+    if (!empty($this->configuration['bg_attach'])) {
+      $bg_attach = $row->getSourceProperty($this->configuration['bg_attach']);
+      $bg_attach_mapping = [
+        'bg-attachment-fixed' => 'bg-fixed',
+        'bg-attachment-scroll' => '',
+      ];
+      foreach ($bg_attach as $bg_attach_item) {
+        $behavior['bg_attachment'] = $bg_attach_mapping[$bg_attach_item['value']];
+      }
+    }
+
+    if (!empty($this->configuration['media_mode'])) {
+      if (!empty($row->getSourceProperty($this->configuration['media_mode']))) {
+        $media_mode_mapping = [
+          'uaqs_bg_img_content_left' => 'col-md-8 col-lg-6',
+          'uaqs_bg_img_content_center' => 'col-md-8 col-lg-6 col-md-offset-2 col-lg-offset-3',
+          'uaqs_bg_img_content_right' => 'col-md-8 col-lg-6 col-md-offset-4 col-lg-offset-6',
+        ];
+      }
+      $behavior['position'] = $media_mode_mapping[$row->getSourceProperty($this->configuration['media_mode'])];
```

Comment:
I think this line should be in the nested if statement.
```suggestion
        $behavior['position'] = $media_mode_mapping[$row->getSourceProperty($this->configuration['media_mode'])];
      }
```